src/OrchardCore.Modules/OrchardCore.ContentLocalization/Controllers/AdminController.cs

```diff
@@ -7,28 +7,24 @@
 using OrchardCore.DisplayManagement.ModelBinding;
 using OrchardCore.DisplayManagement.Notify;
 
-namespace OrchardCore.ContentLocalization.Controllers
-{
-    public class AdminController : Controller, IUpdateModel
-    {
+namespace OrchardCore.ContentLocalization.Controllers {
+    public class AdminController : Controller, IUpdateModel {
         private readonly IContentManager _contentManager;
         private readonly IContentLocalizationManager _contentLocalizationManager;
         private readonly INotifier _notifier;
         private readonly IHtmlLocalizer<AdminController> _localizer;
 
         public IHtmlLocalizer T { get; }
 
-        public AdminController(IContentManager contentManager, INotifier notifier, IContentLocalizationManager localizationManager, IHtmlLocalizer<AdminController> localizer)
-        {
+        public AdminController(IContentManager contentManager, INotifier notifier, IContentLocalizationManager localizationManager, IHtmlLocalizer<AdminController> localizer) {
             _contentManager = contentManager;
             _notifier = notifier;
             _localizer = localizer;
             _contentLocalizationManager = localizationManager;
             T = localizer;
         }
-        
-        public async Task<IActionResult> Localize(string contentItemId, string targetCulture)
-        {
+
+        public async Task<IActionResult> Localize(string contentItemId, string targetCulture) {
```

Comment:
Please update all your changes to follow the coding style (curly braces on new lines)
There is an editorsettings file in the solution that VS should pick up automatically.

Reply:
Ok will do, I was using vscode on linux today.